### PR TITLE
[ES] Clean up `osu! originals`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_originals/es.md [hunk 1]
```diff
@@ -11,22 +11,22 @@ Todos los osu! originals que han sido licenciados por el juego están autorizado
...
 | [nekodex - welcome to osu!](https://soundcloud.com/nekodex/welcome-to-osu)[^fa] | Antiguo tema de osu! |
-| [Peter Lambert - osu! tutorial](https://osu.ppy.sh/beatmapsets/3756) | Antigua canción del tutorial |
+| [Peter Lambert - osu! tutorial](https://osu.ppy.sh/beatmapsets/3756) | Antigua canción de tutorial |
 
 ### Lanzamientos exclusivos de artistas destacados
 
 #### [cYsmix](https://osu.ppy.sh/beatmaps/artists/2)
 
-*Para la publicación de la noticia, véase: [Mapping With Rewards (Sep. 2015)](https://blog.ppy.sh/post/129936251068/mapping-with-rewards-sep-2015)*
+*Para la publicación de noticia, véase: [Mapping With Rewards (Sep. 2015)](https://blog.ppy.sh/post/129936251068/mapping-with-rewards-sep-2015)*
```
Comment (Repflez) [inline wiki/osu!_originals/es.md:29]:
```suggestion
*Para la publicación de la noticia, véase: [Mapping With Rewards (Sep. 2015)](https://blog.ppy.sh/post/129936251068/mapping-with-rewards-sep-2015)*
```
Previous was correct.

File: wiki/osu!_originals/es.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Todos los osu! originals que han sido licenciados por el juego están autorizado
 
 #### [happy30](https://osu.ppy.sh/beatmaps/artists/317)
 
-*Para la publicación de la noticia, véase: [New Featured Artist: happy30](https://osu.ppy.sh/home/news/2022-11-02-new-featured-artist-happy30)*
+*Para la publicación de noticia, véase: [New Featured Artist: happy30](https://osu.ppy.sh/home/news/2022-11-02-new-featured-artist-happy30)*
```
Comment (Repflez) [inline wiki/osu!_originals/es.md:48]:
```suggestion
*Para la publicación de la noticia, véase: [New Featured Artist: happy30](https://osu.ppy.sh/home/news/2022-11-02-new-featured-artist-happy30)*
```
Previous was correct.

File: wiki/osu!_originals/es.md [hunk 3]
```diff
@@ -54,9 +54,9 @@ Todos los osu! originals que han sido licenciados por el juego están autorizado
 
 #### [James Landino](https://osu.ppy.sh/beatmaps/artists/39)
 
-*Para la publicación de la noticia, véase: [New Featured Artist: James Landino](https://osu.ppy.sh/home/news/2019-05-31-new-featured-artist-james-landino)*
+*Para la publicación de noticia, véase: [New Featured Artist: James Landino](https://osu.ppy.sh/home/news/2019-05-31-new-featured-artist-james-landino)*
```
Comment (Repflez) [inline wiki/osu!_originals/es.md:57]:
```suggestion
*Para la publicación de la noticia, véase: [New Featured Artist: James Landino](https://osu.ppy.sh/home/news/2019-05-31-new-featured-artist-james-landino)*
```
Previous was correct.

File: wiki/osu!_originals/es.md [hunk 4]
```diff
@@ -79,23 +79,23 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
 
 #### [kiraku](https://osu.ppy.sh/beatmaps/artists/101)
 
-*Para la publicación de la noticia, véase: [New Featured Artist: kiraku](https://osu.ppy.sh/home/news/2020-09-26-new-featured-artist-kiraku)*
+*Para la publicación de noticia, véase: [New Featured Artist: kiraku](https://osu.ppy.sh/home/news/2020-09-26-new-featured-artist-kiraku)*
```
Comment (Repflez) [inline wiki/osu!_originals/es.md:82]:
```suggestion
*Para la publicación de la noticia, véase: [New Featured Artist: kiraku](https://osu.ppy.sh/home/news/2020-09-26-new-featured-artist-kiraku)*
```
Previous was correct.

File: wiki/osu!_originals/es.md [hunk 5]
```diff
@@ -79,23 +79,23 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
...
 
 | Canción |
 | :-: |
 | [kiraku - connection](https://osu.ppy.sh/beatmaps/artists/tracks?artist=kiraku&query=connection) |
 
 #### [Kitazawa Kyouhei](https://osu.ppy.sh/beatmaps/artists/165)
 
-*Para la publicación de la noticia, véase: [New Featured Artist: Kitazawa Kyouhei](https://osu.ppy.sh/home/news/2021-05-15-new-featured-artist-kitazawa-kyouhei)*
+*Para la publicación de noticia, véase: [New Featured Artist: Kitazawa Kyouhei](https://osu.ppy.sh/home/news/2021-05-15-new-featured-artist-kitazawa-kyouhei)*
```
Comment (Repflez) [inline wiki/osu!_originals/es.md:90]:
```suggestion
*Para la publicación de la noticia, véase: [New Featured Artist: Kitazawa Kyouhei](https://osu.ppy.sh/home/news/2021-05-15-new-featured-artist-kitazawa-kyouhei)*
```
Previous was correct.

File: wiki/osu!_originals/es.md [hunk 6]
```diff
@@ -121,7 +121,7 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
 
 #### [ZxNX](https://osu.ppy.sh/beatmaps/artists/288)
 
-*Para la publicación de la noticia, véase: [New Featured Artist: ZxNX](https://osu.ppy.sh/home/news/2022-07-30-new-featured-artist-zxnx)*
+*Para la publicación de noticia, véase: [New Featured Artist: ZxNX](https://osu.ppy.sh/home/news/2022-07-30-new-featured-artist-zxnx)*
```
Comment (Repflez) [inline wiki/osu!_originals/es.md:124]:
```suggestion
*Para la publicación de la noticia, véase: [New Featured Artist: ZxNX](https://osu.ppy.sh/home/news/2022-07-30-new-featured-artist-zxnx)*
```
Previous was correct.

File: wiki/osu!_originals/es.md [hunk 7]
```diff
@@ -105,7 +105,7 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
 
 #### [yuki.](https://osu.ppy.sh/beatmaps/artists/4)
 
-*Para la publicación de la noticia, véase: [Community Mapping Contest #1 - yuki. (Nadeshiko Sensation)](https://osu.ppy.sh/home/news/2016-09-28-community-mapping-contest-1-yuki-nadeshiko)*
+*Para la publicación de noticia, véase: [Community Mapping Contest #1 - yuki. (Nadeshiko Sensation)](https://osu.ppy.sh/home/news/2016-09-28-community-mapping-contest-1-yuki-nadeshiko)*
```
Comment (Repflez) [inline wiki/osu!_originals/es.md:108]:
```suggestion
*Para la publicación de la noticia, véase: [Community Mapping Contest #1 - yuki. (Nadeshiko Sensation)](https://osu.ppy.sh/home/news/2016-09-28-community-mapping-contest-1-yuki-nadeshiko)*
```
Previous was correct.

File: wiki/osu!_originals/es.md [hunk 8]
```diff
@@ -79,23 +79,23 @@ Estas canciones se [anunciaron](https://osu.ppy.sh/home/news/2023-02-01-twin-tri
...
+*Para la publicación de noticia, véase: [New Featured Artist: Kitazawa Kyouhei](https://osu.ppy.sh/home/news/2021-05-15-new-featured-artist-kitazawa-kyouhei)*
 
 | Canción |
 | :-: |
 | [Kitazawa Kyouhei - TRUE-QUEEN](https://www.youtube.com/watch?v=s4FcjckW0NE) |
 
 #### [Rabbit House](https://osu.ppy.sh/beatmaps/artists/242)
 
-*Para la publicación de la noticia, véase: [New Featured Artist: Rabbit House](https://osu.ppy.sh/home/news/2022-02-23-new-featured-artist-rabbit-house)*
+*Para la publicación de noticia, véase: [New Featured Artist: Rabbit House](https://osu.ppy.sh/home/news/2022-02-23-new-featured-artist-rabbit-house)*
```
Comment (Repflez) [inline wiki/osu!_originals/es.md:98]:
```suggestion
*Para la publicación de la noticia, véase: [New Featured Artist: Rabbit House](https://osu.ppy.sh/home/news/2022-02-23-new-featured-artist-rabbit-house)*
```
Previous was correct.

**Review by Repflez — CHANGES_REQUESTED** [review]

**Review by Repflez — APPROVED** [review]
LGTM

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**